docs/pypi.md

```diff
@@ -71,7 +71,11 @@ javascript"](./using_pyodide_from_javascript.html) a complete example would be,
   <meta charset="utf-8">
 </head>
 <body>
-  <script type="text/javascript" src="https://pyodide.cdn.iodide.io/pyodide.js"></script>
+  <script type="text/javascript">
+      // set the pyodide files URL (packages.json, pyodide.asm.data etc)
+      window.languagePluginUrl = 'https://pyodide-cdn2.iodide.io/v0.14.3/full/';
```

Comment:
The reason to explicitly set the version here as opposed to using `/latest/full` is to avoid redirects for each downloaded package. These can be then updated automatically on release with `bumpversion`.